decidim-core/config/locales/ca.yml

```diff
@@ -126,6 +126,14 @@ ca:
         back_home: Tornar a l'inici
         content_doesnt_exist: Aquesta adreça és incorrecta o ha estat eliminada.
         title: No s'ha trobat la pàgina que busques
+    events:
+      email_event:
+        email_greeting: Hola %{user_name},
+        email_intro: 'Hi ha hagut una actualització a "%{resource_title}". Es pot veure d''aquesta pàgina:'
```

Comment:
These translations make no sense

Reply:
AFAIK, this translation is produced as suggestion by Microsoft Translator at Crowdin.
In my experience, Google Translate at Crowdin is usually a better option. 
In this case it offers:
'S'ha produït una actualització de "% {resource_title}". Podeu veure-ho des d'aquesta pàgina:'

Reply:
Yep, that sounds much better than the previous one, but I think @josepjaume had some reasons to use Microsoft Translator. Let's see if he can chime in here :)

Reply:
Sorry about that. It looked like Microsoft Translator provided better translations but in the end I think it's not. Reverting to Google Translate!

A few weeks ago, I configured crowdin in such a way that it exports all translations now, regardless of their approval - this ends up with sometimes quite incorrect translations, but I think it's better than resorting to english when it doesn't. It also encourages the community to help!